.vscodeignore

```diff
@@ -3,9 +3,10 @@
 .vscode/**/*
 .vscode-test/**
 .gitignore
-src/**
```

Comment:
I believe you still need to ignore the whole `src` directory apart from js files. IMO the correct pattern is
```
src
!src/**/*.js
```

Reply:
I think this pattern [`!src/**/*.js`](https://github.com/iTOYS/vscode-react-native/blob/5e5483fcdd03c78edc94f901c5135af2a7b028fe/.vscodeignore#L9) woks fine

Reply:
Not ignoring whole `src` dir basically means that all source ts files will appear in final vsix package. I don't think this is expected behavior